lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -421,7 +421,7 @@ If ($state -eq "present") {
 	}
 
 	If ($insertbefore -and $insertafter) {
-		Add-Warning('Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"');
+		Add-Warning $result 'Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"'
```

Comment:
I am torn for the options though. We don't do it for Exit-Json or Fail-Json, and this one is in the same category.